StreamConsumer/README.md

```diff
@@ -439,3 +585,265 @@ Rapid pro requires a specific format for phone numbers and it's better to change
+              "sms_body_medical_discharge": "$.source.message.sms_body_medical_discharge"
+            },
+            "urns": "$.urn.result"
+          }
+        },
+        "output_map": {
+          "status_code": "$.status_code",
+          "all": "$.reason"
+        }
+      }
+    }
+  ]
+}
+```
+
+```json
+{
+    "id": "tb_11",
+    "name": "TB 11 / TB patient visit at health facility / Follow up Enroll",
+    "kafka_subscription": {
+        "topic_pattern": "TBForm*",
+        "topic_options": {
+            "masking_annotation": "@aether_masking",
+            "masking_levels": ["public", "private"],
+            "masking_emit_level": "public",
+            "filter_required": false
+        }
+    },
+    "error_handling": {
+        "error_topic": "log_tb_11",
+        "log_failure": true,
+        "log_success": true
+
+    },
+    "const": {
+        "rapidpro_flow": "https://api.textit.in/api/v2/flow_starts.json",
+        "rapidpro_auth_header": "Token API_KEY_GOES_HERE",
+        "rapidpro_flow_id__enroll": "c4368cd9-b92e-4020-bbe8-2a20be4b702c",
+        "aether_url": "https://eha-data.org/htidev/kernel/entities.json",
+        "form_id_sample_registration": "TB_suspect_registration",
+        "aether_auth": {"user": "your aether username", "password": "your aether password"}
+    },
+    "stages" : [
+        {
+            "name": "patient",
+            "type": "restcall",
+            "id": "default",
+            "transition": {
+                "input_map": {
+                    "url": "$.const.aether_url",
+                    "method": "GET",
+                    "basic_auth": "$.const.aether_auth",
+                    "query_params": {
+                        "ordering": "-modified",
+                        "payload___id": "$.const.form_id_sample_registration",
+                        "payload__patient_id": "$.source.message.patient_id"
+                    }
+                },
+                "output_map":{
+                    "status_code": "$.status_code",
+                    "contact_number": "$.json.results[0].payload.contact_number"
+                }
+            }
+        },
+        {
+            "name": "urn",
+            "type": "jscall",
+            "id": "urn",
+            "transition": {
+                "input_map": {
+                    "base": "$.patient.contact_number"
+                },
+                "output_map": {
+                    "result": "$.result"
+                }
+            }
+        },
+        {
+            "name": "rapidpro",
+            "type": "restcall",
+            "id": "default",
+            "transition": {
+                "input_map": {
+                    "url": "$.const.rapidpro_flow",
+                    "method": "POST",
+                    "headers": {"Authorization": "$.const.rapidpro_auth_header"},
+                    "json_body": {
+                        "flow": "$.const.rapidpro_flow_id__enroll",
```

Comment:
This assumes you'll use the same campaign for TB reminders and HIV Rural reminders. Would be good to check that with the partner. If so a person couldn't be in TB and HIV reminder campaigns at the same time (because a person / phone number can only be enrolled in a campaign once.)

Reply:
My assumption is in fact that we will use different campaigns. I will check with the partners